ChangeLog.md

```diff
@@ -1,5 +1,11 @@
 # ChangeLog
 
+## [Unreleased](https://github.com/algolia/algoliasearch-client-go/compare/2.28.0...master) (2019-07-08)
+
+### Changes
+
+- **added:** Implement Index.Exists ([21b33e4](https://github.com/algolia/algoliasearch-client-go/commit/21b33e4))
```

Comment:
Because we now generates the ChangeLog at release time using our internal tool `polyglot`, you should not need an extra commit to update the ChangeLog. Simply add a line at the end of your commit message with `[changelog]`. More details there: https://github.com/algolia/polyglot#polyglot-changelog. Note that this is not made explicit and public yet but we're planning on doing it soon on all API clients.

Reply:
Reverted, and added the line in one of my commit.